specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-02-01-preview/machineLearningServices.json

```diff
@@ -1970,6 +1970,324 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/managedNetwork": {
+      "get": {
```

Comment:
please check all the failures flagged here and address them - [https://github.com/Azure/azure-rest-api-specs/pull/22588/checks?check_run_id=11320418747](https://github.com/Azure/azure-rest-api-specs/pull/22588/checks?check_run_id=11320418747)